.travis.yml

```diff
@@ -10,9 +10,9 @@ python:
 os:
     - linux
 stages:
+  - lint
```

Comment:
thanks but can you revert this?

Reply:
@auvipy I've done that, why though?

Reply:
we used to run lint first for many years. but later we find out that, many initial contributors/comparatively irregular contributors do not get the benefit of running their code through the unit & integration test for lint errors and the don't get the benefit of early feedback loop on their actual code changes. so we come to the conclusion that we should let the unit and integration tests first.